cromwell-helm/values.yaml

```diff
@@ -43,75 +43,5 @@ cromwell:
     dir: /etc/conf
     file: papi.conf
 
-jobmanager:
```

Comment:
Question for reviewers: This can now be removed right? I wasn't able to find cromwell-helm references for this and hence removed it but I am not sure if it is used else where.

Reply:
Yeah, if it's not used here seems fine to remove. Might want to double-check in Leo that it isn't trying to override any of it? I doubt it but doesn't hurt to be safe.

Reply:
I wasn't able to find any job manager reference in [CromwellAppInstall](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellAppInstall.scala).